app/pages/sections/order-items/associated-services/routes.js

```diff
@@ -0,0 +1,11 @@
+import express from 'express';
+import { logger } from '../../../../logger';
+import { withCatch } from '../../../../helpers/routes/routerHelper';
+
+const router = express.Router({ mergeParams: true });
+
+export const associatedServicesRoutes = (authProvider) => {
+  router.get('/', authProvider.authorise({ claim: 'ordering' }), withCatch(logger, authProvider, async (req, res) => res.send('Additional services page')));
```

Comment:
Doesn't really matter as you are probably going to change this shortly but... Associated services?

Reply:
Oops, yes. It should be Associated services.